scripts/scil_find_script.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+Search through all of SCILPY scripts and their docstrings. The output of the
+search will be the intersection of all provided keywords, found either in the
+script name or in its docstring.
+"""
+
+from argparse import ArgumentParser, RawTextHelpFormatter
+import ast
+import inspect
+import pathlib
+import re
+
+import numpy as np
+import scilpy
+from scilpy.io.utils import add_verbose_arg
+
+RED = '\033[31m'
+BOLD = '\033[1m'
+ENDC = '\033[0m'
+
+
+def _build_arg_parser():
+    parser = ArgumentParser(description=__doc__,
+                            formatter_class=RawTextHelpFormatter)
+    parser.add_argument('keywords', nargs='+',
+                        help="Search keywords")
+    add_verbose_arg(parser)
+    return parser
+
+
+def main():
```

Comment:
Is there an agreed convention on the order of the methods in the scripts, e.g.  `_build_arg_parse`r and `main` always first, then the rest of methods, or else are the majority of the scripts making these two be last methods?

Reply:
I would guess that `build_arg_parser` should come first, but I don't think there is any convention for the rest

Reply:
OK. I guess having one would improve the readability of the scripts, but it should not be a show-stopper.